packages/web-components/src/components/va-on-this-page/va-on-this-page.tsx

```diff
@@ -90,17 +90,17 @@ export class VaOnThisPage {
 
     return (
       <nav aria-labelledby="on-this-page">
-        <dl>
-          <dt id="on-this-page">{i18next.t('on-this-page')}</dt>
-          <dd role="definition">
-            {h2s.map(heading => (
+        <ul>
+          <li id="on-this-page">{i18next.t('on-this-page')}</li>
+          {h2s.map(heading => (
+            <li>
               <a href={`#${heading.id}`} onClick={handleOnClick}>
                 <va-icon icon="arrow_downward"></va-icon>
                 {heading.innerText}
               </a>
-            ))}
-          </dd>
-        </dl>
+            </li>
```

Comment:
Are there tests that this will affect in vets-website?

Reply:
There is one test that looks for this component: vets-website/src/applications/sco/tests/accessibility.cypress.spec.js, but the test is structured in such a way that it should still work with this re-write.

It's worth noting that there are currently 5 pages that have "fake component" versions of the va-on-this-page that all still use the dl/dd combo to display links and should probably be converted:
1. vets-website/src/applications/combined-debt-portal/debt-letters/components/DebtDetailsOnThisPageLinks.jsx
2. vets-website/src/applications/combined-debt-portal/debt-letters/components/OnThisPageLinks.jsx
3. vets-website/src/applications/combined-debt-portal/medical-copays/components/OnThisPageDetails.jsx
4. vets-website/src/applications/combined-debt-portal/medical-copays/components/OnThisPageOverview.jsx
5. vets-website/src/applications/combined-debt-portal/medical-copays/components/OnThisPageStatements.jsx

I'll write these up in a separate ticket for after this component ships.